elements/pl-pivot-table/pl-pivot-table-double.mustache

```diff
@@ -0,0 +1,106 @@
+{{#question}}
+        <script>
+            var uuid = "{{uuid}}";
+            var num_row_dropzone = parseInt("{{num_row_dropzone_val}}");
+            var num_index = 2;
+            var is_multicol = "True" === "{{is_multicol}}";
+        </script>
+        <div class="container column-box">
+            <h2>Choose answer column here</h2>
+            {{#column_set}}
+            <div class="row col-set" id="{{uuid}}" data-order="{{order_col}}">
+                {{#column}}
+                <div class="col-md-{{{width}}} col-cell">{{{inner_html}}}</div>
+                {{/column}}
+                {{#is_ellipsis}}
+                <div class="col-md-{{{width}}} col-cell">...</div>
+                {{/is_ellipsis}}
+            </div>
+            {{/column_set}}
+        </div>
+
+        <div class="container index-box">
+            <h2>Choose answer index here</h2>
+            <div class="row">
+                {{#indice_set}}
+                <div class="col-md-1"{{{calibration}}}>
+                    <div class="index-set" id="{{uuid}}" data-order="{{order_index}}">
+                        {{#index}}
+                        <div class="row">
+                            <div class="col-md-12 index-cell" style="height: 30px;">{{{inner_html}}}</div>
```

Comment:
please do not use inline styles at all.  and definitely do not use fixed heights like"30px", which breaks if the user resizes their font or changes their zoom level.  use the `-py` and `-my` and similar classes in bootstrap if you need to adjust vertical spacing or height.

Reply:
This is resolved